### ISSUE TITLE
try to use MLFPN on exist SSD model, but the mAP did not reach as expected...?

### ISSUE THREAD

**Issue by ximitiejiang** [qijiezhao/M2Det#48]
hello, I tried to move MLFPN on exist SSD model which has been proved to reach mAP0.78 on VOC07+12, but when I add MLFPN to this SSD(vgg16, pretrained model from caffe, together with mean/std setting.) the mAP can only reach to 0.57 when i trained 80 epochs with below setting:
1) 2x1080Ti
2) 4 imgs per GPU (also tried 8 imgs per GPU but not help to the loss)
3) lr = [0.002, 0.0002, 0.0002], epoch = [40,80,120]
i trained the model  for 120 epochs but the loss reach stable after 80 epoch, no use for increase from 4 img per GPU to 8 img per GPU(GPU cache has reach to 10G), no use for increase lr to 0.001, currenly the mAP on VOC is 0.57, could you give any suggestion about my training?
by the way i did not use softnms, but the affect should not be so big...
